openstack/compute/v2/extensions/quotasets/results.go

```diff
@@ -55,6 +55,68 @@ type QuotaSet struct {
 	ServerGroupMembers int `json:"server_group_members"`
 }
 
+// QuotaDetailSet is detail on both operational limits that allow for control
+// of compute usage and also currently measured allocation against those limits.
```

Comment:
perhaps: "QuotaDetailSet represents details of both operational limits of compute resources and the current usage of those resources"